compute/test/createInstanceTemplates.test.js

```diff
@@ -34,7 +34,8 @@ const createInstance = async (projectId, zone, instanceName) => {
         {
           initializeParams: {
             diskSizeGb: '250',
-            sourceImage: 'projects/debian-cloud/global/images/family/debian-10',
+            sourceImage:
```

Comment:
used the same path as in https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/compute/create-instance-templates/createTemplateFromInstance.js